joern-cli/frontends/pysrc2cpg/src/test/scala/io/joern/pysrc2cpg/dataflow/DataFlowTests.scala

```diff
@@ -473,5 +475,54 @@ class RegexDefinedFlowsDataFlowTests
       snk.reachableByFlows(src).size shouldBe 2
     }
   }
+  "Import statement with method ref sample four" in {
+    val controller =
+      """
+        |from django.contrib import admin
+        |from django.urls import path
+        |from django.conf.urls import url
+        |from .views import all_page
+        |
+        |urlpatterns = [
+        |    url(r'allPage', all_page)
+        |]
+        |""".stripMargin
+    val views =
+      """
+        |def all_page(request):
+        |	print("All pages")
+        |""".stripMargin
+    val cpg = code("")
```

Comment:
You can simply do:
```
code(controller, Seq("controller", "urls.py").mkString(File.separator))
       .moreCode(views, Seq("controller", "views.py").mkString(File.separator))
```
Otherwise, it "creates" three files and one is empty.

Reply:
done